.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: Gytheio CI
+
+on:
+  pull_request:
+    branches:
+      - master
+      - SP/**
+      - HF/**
+      - ACS-**
+      - feature/**
+  push:
+    branches:
+      - master
+      - SP/**
+      - HF/**
+      - ACS-**
+      - feature/**
+  schedule:
+    - cron: '0 5 * * 3'
+  workflow_dispatch:
+
+env:
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  GIT_USERNAME: ${{ secrets.BOT_GITHUB_USERNAME }}
+  GIT_PASSWORD: ${{ secrets.BOT_GITHUB_TOKEN }}
+  AWS_REGION: eu-west-1
+
+jobs:
+  veracode:
+    name: "Source Clear Scan (SCA)"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.35.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.35.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/veracode@v1.35.0
+        with:
+          srcclr-api-token: ${{ secrets.SRCCLR_API_TOKEN }}
+
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.35.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.35.0
+      - name: "Remove SNAPSHOT artifacts"
+        run: find "${HOME}/.m2/repository/" -type d -name "*-SNAPSHOT*" | xargs -r -l rm -rf
+      - name: "Install ImageMagick to parse PDFs"
+        run: |
+          sudo apt-get update
+          sudo apt-get install -q -y  imagemagick ghostscript ffmpeg
+          sudo sed -i '/PDF/s/none/read|write/' /etc/ImageMagick-6/policy.xml
+      - name: "Configure AWS credentials"
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ env.AWS_REGION }}
+      - name: "Build and test"
+        run: bash _ci/build.sh
+
+  release:
+    name: "Release"
+    needs: [veracode, build]
+    runs-on: ubuntu-latest
+    if: >
+      !failure() &&
```

Comment:
Is this `!failure()` condition actually needed? I can't see any way of skipping the build and veracode jobs.